### PR TITLE
Update/tabs

### PR DESCRIPTION
**Pull request by kfarr2**
Wrapped everything up and stuck it into a tabbed container.

### CODE REVIEW

**Comment by kfarr2** [comment]
Dangit. Forgot to run flake8 and isort. Gimme a minute.

File: hotline/notifications/views.py [hunk 1]
```diff
@@ -29,6 +36,19 @@ def create(request):
 
 @permissions.is_active
 def list_(request):
+    # all that awesome tabs stuff
```
Comment (mdj2) [inline hotline/notifications/views.py]: We've got this same logic in two places. Can it be refactored? Maybe as a method on the user model?

Comment (kfarr2) [inline hotline/notifications/views.py]: Just saw this. I'll work on this tomorrow morning.

**Comment by mdj2** [comment]
I'm gonna merge this in quickly for the demo this morning (so don't rebase it).

**Pull request merged**